src/lib/feature-flags.ts

```diff
@@ -5,10 +5,22 @@ export interface FeatureFlagDefinition {
 	default: boolean;
 }
 
-// eslint-disable-next-line @typescript-eslint/no-empty-object-type
-export interface FeatureFlags {}
+export const FEATURE_FLAGS_DEFINITION = {
```

Comment:
Moving forward this should be the only change we need to add or remove feature flags